cmd/root.go

```diff
@@ -162,17 +162,19 @@ func NewDataCommand() *cobra.Command {
 	return command
 }
 
-func (r *RootOptions) initConfig(_ *cobra.Command, _ []string) error {
+func (r *RootOptions) initConfig(cmd *cobra.Command, _ []string) error {
 	r.configAccessor = viper.NewAccessor(config.Options{
 		StrictMode:  true,
 		SearchPaths: []string{r.cfgFile},
 	})
 
+	// persistent flags were initially bound to the root command so we must bind to the same command to avoid
+	r.configAccessor.InitializePflags(cmd.PersistentFlags())
```

Comment:
Are there no test changes required for this change

Reply:
no we don't have test fro root

Reply:
Is it too difficult to add one for this function. If yes then we can defer it or else would be good to improve on this aspect